lib/mongo/cursor.ex

```diff
@@ -59,7 +59,11 @@ defmodule Mongo.Cursor do
 
           case Mongo.get_more(conn, coll, cursor, opts) do
             {:ok, %{cursor_id: cursor, docs: []}} ->
-              {:halt, state(state, cursor: cursor)}
+              if opts[:tailable_cursor] == true do
```

Comment:
Drop the `== true`? Or move `opts[:tailable_cursor]` into a variable and make it a guard on the case?